### ISSUE TITLE
[Legitimate Site Blocked] murakamiflowers.kaikaikiki.com

### ISSUE THREAD

**Issue by Mizonal** [MetaMask/eth-phishing-detect#7275]
https://murakamiflowers.kaikaikiki.com/

This site is true site of Murakami.Flowers.
please check Takashi's twitter or instagram.
https://www.instagram.com/takashipom/
https://twitter.com/takashipom

**Comment by nsfx84** [comment]
Yes this is a legit site. Not sure why it's been blacklisted all of a sudden.

**Comment by Tasarvo** [comment]
Please please unblock this, this is the OFFICIAL site, you can see its official from the official Opensea account, the reveal is happening soon i think, Takashi Murakami has his exhibition happening on the 11th, please expediate this!  The reason there are tons of fake sites is because of the HUG following Takashi Murakami has internationally, the largest artist launching his own priject, the previous being the deisgn of CLone X that Nike purchased!

**Comment by TheF604** [comment]
Chiming in my +1
This is the legit site and needs to be unblocked.

**Comment by devenblake** [comment]
Duplicate of #7260.
